content/blog/rails-dynamic-translation-constants.md

```diff
@@ -0,0 +1,156 @@
++++
+title = "Dynamic Translations in Rails: Why Constants Can’t Adapt to Locale Changes and How to Fix It?"
+slug = "rails-i18n"
```

Comment:
i believe this needs to change to align with Blog Title as this become the url of the blog
```suggestion
slug = "rails-dynamic-translation-constants"
```